string path
string image_name
---
int32 status